index.php

```diff
@@ -9,10 +9,60 @@
     <div class="wrapper">
         <div id="logo-bar">
             <?php include 'logo-include.php'; ?>
-            <h1> Flock Information Database Home </h1>
-        </div>        
+            <h1> Home </h1>
+        </div>
+
         <div class="content">
-            <h2> Stick a login form here </h2>
+            <table id="register-login">
+            <tr>
+            <td><form method="post" action="index.php">
+                <table id="inner-register">
```

Comment:
I think we should give the id to the form instead. It'll make it easier to make any edits to the form later. Accessing the table in the css will be just as easy, it will be 
```css
#register table {
  //whatever css
}
```